### ISSUE TITLE
Feature Request: Changing display font & margin

### ISSUE THREAD

**Issue by sirpooya** [ayltai/Developers-Excuses#12]
This is an amazing screen-saver, however it would be much convenient if we could choose display font for quotes and even margin of page so for long quotes, the length in each line doesn't get wide & harder to read.

Thanks in advance

**Comment by ayltai** [comment]
It seems to me that would become too many configurations for an average user and those configurations are too low-level. I would suggest:
1. adding one more option with 2 or 3 predefined font styles to choose from, and
2. letting users to update a text file to change the margins and all other low-level configurations.

**Comment by sirpooya** [comment]
I totally agree, it would be too many configurations in the UI, so is it possible for you to hardcode them in project so I could build the project myself? and as you suggested:

1. Predefined font could be changeable in code.
2. Text View width could be defined as a fluid value based on View Width (percentage)
